man/ostree.repo-config.xml

```diff
@@ -76,7 +76,7 @@ Boston, MA 02111-1307, USA.
     <variablelist>
       <varlistentry>
         <term><varname>mode</varname></term>
-        <listitem><para>One of <literal>bare</literal>, <literal>bare-user</literal> or <literal>archive-z2</literal>.  </para></listitem>
+        <listitem><para>One of <literal>bare</literal>, <literal>bare-user</literal> or <literal>archive-z2</literal> (Note that <literal>archive</literal> is used everywhere else.)</para></listitem>
```

Comment:
Nit: shouldn't that 'N' be lower case?

Also, let's put that notice in `man/ostree-init.xml` as well?

Reply:
Sure, done ⬇️